tests/console.spec.ts

```diff
@@ -1,12 +1,16 @@
 import { test, expect, Page } from '@playwright/test';
 import shortid from 'shortid';
-import { generateVarName, makeTableCommand, typeInMonaco } from './utils';
+import { generateVarName, makeTableCommand, pasteInMonaco } from './utils';
 
 let page: Page;
+let consoleInput: Locator;
 
 test.beforeAll(async ({ browser }) => {
   page = await browser.newPage();
   await page.goto('');
+
+  consoleInput = page.locator('.console-input');
+  await consoleInput.click();
```

Comment:
Are these clicks fallback for when "pasting" in Firefox? If so, I would put it in the `pasteInMonaco` function instead.

Doesn't look like the `typeInMonaco` function requires the click, but that seems to be a method that is really just "type in whatever is active"

Can be a follow up ticket to fix these utils though. If you open another ticket just re-request and I'll approve